src/App.tsx

```diff
@@ -28,23 +26,23 @@ function App() {
       <Route
         exact
         path="/events/new"
-        component={(props) => <EventCreationPage mode="new" {...props} />}
+        component={(props) => <MapPage mode="newEvent" {...props} />}
```

Comment:
these should use the enum strings as well (after importing `graphql/queries/maps`)
```suggestion
        component={(props) => <MapPage mode={MapModes.newEvent} {...props} />}
```